SonarQube.Client/Models/ServerExclusions.cs

```diff
@@ -19,6 +19,7 @@
  */
 
 using System;
+using Newtonsoft.Json;
 
 namespace SonarQube.Client.Models
```

Comment:
This PR fixes 2 issues but I didn't want to update the submodule twice for such a minor change.